src/context_store/sync/outbox_worker.py

```
@@ -0,0 +1,136 @@
+            mids = [str(e.memory_id) for e in sync_events]
+            try:
+                memories = await self._storage.get_memories_batch(mids)  # type: ignore[attr-defined]
+                found_ids = {str(m.id) for m in memories}
+                orphan_ids = [str(e.id) for e in sync_events if str(e.memory_id) not in found_ids]
+                if memories:
+                    await self._graph_sync.bulk_merge_memories(memories)
+                completed_ids.extend(orphan_ids)
+                completed_ids.extend(
+                    str(e.id) for e in sync_events if str(e.memory_id) in found_ids
+                )
+            except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)